app/controllers/api/moves_controller.rb

```diff
@@ -9,7 +9,10 @@ def index
     end
 
     def csv
-      send_file(Moves::Exporter.new(moves).call, type: 'text/csv', disposition: :inline)
+      # NB: this method should behave the same irrespective of api version / includes etc
+      csv_moves = Moves::Finder.new(filter_params, current_ability, params[:sort] || {},
+                                    [:from_location, :to_location, { profile: :documents }, person: %i[gender ethnicity]]).call
```

Comment:
Might be worth splitting out includes for readability e.g.

```ruby
csv_includes = [:from_location, :to_location, { profile: :documents }, person: %i[gender ethnicity]]
```

That probably provides enough clarity to make the comment on line 12 redundant too.

Reply:
added `csv_includes` as a constant